### DISCUSSION TITLE
RightSizing Recommendation and the Cost Saving Logic

### DISCUSSION THREAD

**Discussion by rohit1dhiman** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#714] [General]
Hi All, I want to customize this dashboard, can some one please help me to guide where is potential   cost saving and right sizing recommendation logic is written for ec2 instances in tao dashboard

**Comment by iakov-gan** [comment]
Hello, 
TAO dashboard uses [Trusted Advisor checks](https://docs.aws.amazon.com/awssupport/latest/user/cost-optimization-checks.html). Please note that Trusted Advisor counts the full cost of underutilized resource as a potential savings, which can be fully applicable if you remove the resource, but not if you need to resize/replace it.

Regarding Right Sizing of EC2 we recommend using Compute Optimizer Dashboard that uses more accurate data from AWS Compute Optimizer Service. These take into account resizing of instances to the optimal type.  AWS Compute Optimizer uses machine learning to best match the CPU and other metrics available on CloudWatch and propose optimal instance type. 
https://catalog.workshops.aws/awscid/dashboards/advanced/compute-optimizer


Please contact your AWS Account team (TAM, SA) to organize a workshop around customization of dashboards for your needs.

**Reply by rohit1dhiman** [reply]
thanks @iakov-aws  for your quick response
